api/answer.py

```
@@ -1242,50 +1384,389 @@ def check_llm_connection(self) -> bool:
+            self.separator = '\t'
+
+    def _query(self, q_info: dict) -> Optional[str]:
+        # 强行关闭清除所有当前活动的 tqdm 进度条
+        try:
+            from tqdm import tqdm
+            for instance in list(tqdm._instances):
+                instance.leave = False
+                instance.clear()
+                instance.close()
+        except Exception:
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)